### PR TITLE
update readme for fix markdown lint

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind bug
/kind documentation

**What this PR does / why we need it**:

fixed markdown lint error by adding missing alternate text for images.

see
```
markdownlint docs --ignore docs/content/en/references --ignore docs/node_modules -c common/config/mdl.json
markdownlint ./README.md -c common/config/mdl.json
./README.md:33:5 MD045/no-alt-text Images should have alternate text (alt text)
```

### CODE REVIEW

**Comment by Xieql** [comment]
@hzxuzhonghu  PTAL,  without addressing this issue, all other PRs will encounter check failures."

**Comment by Xieql** [comment]
Fixes #514

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ It provides powerful capabilities for multi-cloud and multi-cluster management,
 ## Architecture
 
 <div  align="center">
```
Comment (zirain) [inline README.md:32]: why not use markdown sytnax?

Comment (Xieql) [reply README.md]: you are right

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**